tools/run-e2e-test.ps1

```diff
@@ -0,0 +1,66 @@
+### This script is a quick way to run the XHarness E2E tests.
+### These test are located in tests/integration-tests and require the Arcade and Helix SDK.
+### To run them, you need to invoke these through MSBuild which makes the process a bit cumbersome.
+### This script should make things easier.
+###
+### Usage: .\run-e2e-test.ps1 -TestProject Apple/SimulatorInstaller.Tests.proj [-SkipBuild]
+
+param (
+    [Parameter(Mandatory = $true)]
+    [string]
+    $TestProject,
+    
+    [switch]
+    $SkipBuild = $false
+)
+
+$repoRoot = "$PSScriptRoot\.."
+
+function Write-Projects {
+    Write-Output "Possible options:"
+    $files = Get-ChildItem -Include *Tests.proj -Recurse -Path "$repoRoot\tests\integration-tests\Android"
+    foreach ($item in $files) {
+        '  - Android\{0}' -f $item.Name
```

Comment:
This will show invalid path if project is in subfolder.

Reply:
Yeah, I didn't know how to handle this part well in PowerShell. We have so far this structure and we actually do not want to go to subfolders.

In bash, I just do `find` with `maxDepth` and then process it. In PowerShell I didn't know how to strip the first part of the absolute path.

Reply:
You can do similar thing in PowerShell:

```
$testRoot = "$repoRoot\tests\integration-tests\"
Get-ChildItem -Recurse -Include *Tests.proj -Path $testRoot | ForEach-Object { $_.FullName.Substring($testRoot.Length) }
```